go/kbfs/libkbfs/cr_chains.go

```diff
@@ -394,6 +397,27 @@ func (cc *crChain) hasSetAttrOp() bool {
 	return false
 }
 
+func (cc *crChain) ensurePath(op op, ptr data.BlockPointer) {
+	if op.getFinalPath().IsValid() {
+		return
+	}
+
+	// Use the same obfuscator for both the node's name and it's
+	// children, because it's too complicated to try to get the real
+	// parent obfuscator from the chain.
```

Comment:
Not sure what it could do if it knew it was at the wrong level -- add a `-wrong` (or maybe `-*` to be more succint) suffix to everything?  I personally think something like that would be more noisy than it's worth, for the very rare cases where the conflicts cause problems.  My plan is to rely on the upcoming `deobfuscate` command, asking for the users help if something needs to be disambiguated.

But I'm definitely open to other thoughts, I could be missing something good here.

Reply:
You're the one that has more experience debugging KBFS, so I'll defer to your judgment here.